### ISSUE TITLE
github_issue_converter.py does not include label, title or status changes in message

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Austin503/support-tools#73]
```
What steps will reproduce the problem?
1. Open an issue which has status, title or label changes, like 
https://code.google.com/p/tortoisegit/issues/detail?id=194

What is the expected output? What do you see instead?
The migrated issue neither show that this issue was merged into issue #101 nor 
that in comment #21 the status was changed to "ThirdPart".
```

Original issue reported on code.google.com by `sstrickr...@googlemail.com` on 11 Apr 2015 at 6:44

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
Original comment by `sstrickr...@googlemail.com` on 11 Apr 2015 at 9:57

Attachments:
- [0001-Handle-changes-of-labels-status-and-show-merged-into.patch](https://storage.googleapis.com/google-code-attachments/support-tools/issue-73/comment-2/0001-Handle-changes-of-labels-status-and-show-merged-into.patch)

**Comment by GoogleCodeExporter** [comment]
Original comment by `chrsm...@google.com` on 26 Apr 2015 at 1:14
- Changed state: **Accepted**
- Added labels: **Component-ExporterScripts**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
It doesn't include "blocked-on", "merged-into", and other types of meta-changes 
as well.
```

Original comment by `chrsm...@google.com` on 4 Jun 2015 at 3:54

**Comment by GoogleCodeExporter** [comment]
```
From the Firebug folk:

> I just realized that also relationships between issues are gone.
> E.g. issue 6260 is the meta-issue for issue 6258, 6259, 6260
> and 6414 (see issue 3050 on Google Code), which is now not tracked anymore.

The original (see the 4 issues it is "blocked on"):
https://code.google.com/p/fbug/issues/detail?id=3050

The original dependent issue (see it is "blocking" #3050):
https://code.google.com/p/fbug/issues/detail?id=6138

The exported source issue:
https://github.com/firebug/firebug/issues/3189

The exported dependent issue:
https://github.com/firebug/firebug/issues/6260
```

Original comment by `chrsm...@google.com` on 4 Jun 2015 at 4:07

**Comment by GoogleCodeExporter** [comment]
```
The issue exporter now shows the following types of metadata:

- Changes to "blocking" or "blocked-on"
- Changes to the issue's status
- Changes to the "merged into" status

It already shows label changes. (Unless I'm missing something from the initial 
bug report.)

As for showing title changes, unfortunately I cannot fix that. The Google 
Takeout dump only has the "changed-to" value when the status changes, and the 
issue's final title. So while it is possible to show title changes, it is a 
semi-moot point since the original title is changed. (And only meaningful if 
there are 3+ title changes.)
```

Original comment by `chrsm...@google.com` on 4 Jun 2015 at 10:20
- Changed state: **Fixed**
